common/src/Microsoft.Azure.IIoT.Hub.Module.Client/src/Default/IoTSdkFactory.cs

```diff
@@ -53,9 +54,9 @@ public sealed class IoTSdkFactory : IClientFactory, IDisposable {
             }
 
             // The runtime injects this as an environment variable
-            var deviceId = Environment.GetEnvironmentVariable("IOTEDGE_DEVICEID");
-            var moduleId = Environment.GetEnvironmentVariable("IOTEDGE_MODULEID");
-            var ehubHost = Environment.GetEnvironmentVariable("IOTEDGE_GATEWAYHOSTNAME");
+            var deviceId = Environment.GetEnvironmentVariable(IoTEdgeVariables.IOTEDGE_DEVICEID);
```

Comment:
The call to get environment variable should be in the class instead of here.

Reply:
@barnstee, as stated in the other comment, `IoTEdgeVariables` is intended as a collection of string literals of IoT Edge runtime environment variable names. I agree that we need a separate configuration class for those values that are coming from IoT Edge runtime as environment variables. But let's do that as a part of a separate PR.